apps/src/MLTrainers.js

```diff
@@ -48,7 +48,7 @@ export function predict(modelData) {
       convertTestValue(
         modelData.featureNumberKey,
         feature,
-        modelData.testData[feature]
+        modelData.testData[feature.replace(/\W/g, '')]
```

Comment:
Since we have this snippet a few times now, can we hoist it to a helper function?

Reply:
Where should it live? It's used in both ai.js and here in MLTrainers.js. Just pick one and export to the other?

Reply:
I'm not too sure about the `/apps` heirarchy, but I think it makes sense for `applab/ai.js` to reach into its parent directory for such a utility function (since it seems we have a variety of "utils" and "common" files there).  It might be cleanest to add this to `apps/src/utils.js` so that we don't create a dependency between the two existing source files, or to create a new `apps/src/aiUtils.js` file?